go.mod

```diff
@@ -120,7 +120,7 @@ require (
 	github.com/russellhaering/goxmldsig v1.4.0
 	github.com/schollz/progressbar/v3 v3.13.1
 	github.com/sethvargo/go-diceware v0.3.0
-	github.com/sirupsen/logrus v1.9.0 // replaced
+	github.com/sirupsen/logrus v1.9.0
```

Comment:
Please edit .github/dependabot.yml too.

Reply:
Thank you, updated.  I also removed `/pkg/sftp` which is no longer replaced as of 40c113b8dad017613c154efaa16997a186eec7fb